src/main/java/com/spartronics4915/frc2020/commands/IndexerCommands.java

```diff
@@ -199,18 +239,14 @@ public Align(Indexer indexer)
 
     public class LoadFromIntake extends SequentialCommandGroup
     {
-        private Indexer mIndexer;
-
-        public LoadFromIntake(Indexer indexer)
+        public LoadFromIntake()
         {
-            mIndexer = indexer;
-
             addCommands(
-                new Align(mIndexer),
-                new EndLaunch(mIndexer), // for safety
-                new WaitForBallHeld(mIndexer),
-                new LoadBallToSlot(mIndexer, 0),
-                new Spin(mIndexer, 1), new InstantCommand(() -> mIndexer.addBalls(1), mIndexer)
+                new Align(),
+                new EndLaunch(), // for safety
+                new WaitForBallHeld(),
+                new LoadBallToSlot(0),
+                new Spin(1), new InstantCommand(() -> mIndexer.addBalls(1), mIndexer)
```

Comment:
hmm.  one of these commands doesn't have a useful name. (hint: the last one)

Reply:
We’ve fixed this on Camden’s repo. We will integrate it today.